src/Elastic/Connection/DummyClient.php

```
@@ -194,13 +194,15 @@ public function delete( array $params ): array {
 	/**
 	 * @see Client::update
 	 */
-	public function update( array $params ): void {
+	public function update( array $params ): array {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$params'. (undefined)

(UnusedFormalParameter)